pandas/tests/io/json/test_pandas.py

```diff
@@ -1192,6 +1188,16 @@ def test_tz_is_utc(self, ts):
         dt = ts.to_pydatetime()
         assert dumps(dt, iso_dates=True) == exp
 
+    def test_tz_is_naive(self):
+        from pandas.io.json import dumps
```

Comment:
is there a reason this is imported here? i c this is done else where, can you fix this.

Reply:
In the tests, we test both our internal ``pandas._libs.json.dumps`` method and ``json.dumps`` (from the stdlib). It's best to import it inside the function to avoid conflicts between the names.